packages/pigeon/pigeons/core_tests.dart

```diff
@@ -102,9 +102,15 @@ abstract class HostIntegrationCoreApi {
   /// Returns an error, to test error handling.
   Object? throwError();
 
-  /// Responds with an error from an async void function.
+  /// Returns an error from a void function, to test error handling.
   void throwErrorFromVoid();
 
+  /// Returns a flutter error, to test error handling.
+  Object? throwFlutterError();
+
+  /// Returns a flutter error from a void function, to test error handling.
+  void throwFlutterErrorFromVoid();
```

Comment:
We don't need two variants of each of the FlutterError versions of the tests, just one. The reason we have two `throw` tests is because the code generated in each case is fundamentally different and so we need to test each, but there's no extra structural difference here that requires two extra tests instead of just one.